pyprep/prep_pipeline.py

```diff
@@ -21,16 +21,17 @@ class PrepPipeline:
         ocular channels are assigned the type 'eog').
     prep_params : dict
         Parameters of PREP which include at least the following keys:
-        - ref_chs : list | 'eeg'
-            - A list of channel names to be used for rereferencing.
-              Can be a str 'eeg' to use all EEG channels.
-        - reref_chs : list | 'eeg'
-            - A list of channel names to be used for line-noise removed, and
-              referenced. Can be a str 'eeg' to use all EEG channels.
-        - line_freqs : array_like
-            - list of floats indicating frequencies to be removed.
-              For example for 60Hz you may specify ``np.arange(60, sfreq / 2, 60)``.
-              Specify an empty list to skip the line noise removal step.
+            - ref_chs : list | 'eeg'
+                - A list of channel names to be used for rereferencing.
+                  Can be a str 'eeg' to use all EEG channels.
+            - reref_chs : list | 'eeg'
+                - A list of channel names to be used for line-noise removed,
+                  and referenced. Can be a str 'eeg' to use all EEG channels.
+            - line_freqs : array_like
+                - list of floats indicating frequencies to be removed.
+                  For example, for 60Hz you may specify
+                  ``np.arange(60, sfreq / 2, 60)``. Specify an empty list to
+                  skip the line noise removal step.
     montage : DigMontage
```

Comment:
Inspecting the [current docs](https://pyprep.readthedocs.io/en/latest/generated/prep_pipeline.PrepPipeline.html), it's clear that something is wrong:

![image](https://user-images.githubusercontent.com/9084751/101208136-fedc8000-3671-11eb-9677-60bce1ca3625.png)


However, your [changes](https://pyprep--43.org.readthedocs.build/en/43/generated/prep_pipeline.PrepPipeline.html#prep_pipeline.PrepPipeline) seem to make a part of the docstr bold that should not be bold:

![image](https://user-images.githubusercontent.com/9084751/101208252-292e3d80-3672-11eb-86c0-d41c31fc1722.png)


--- 

I am not sure, but would it have been sufficient to just add a blank line right after line 23, instead of increasing the indentation level?

Reply:
Yep, that did it. Thanks!